[SEC-13439] Update Signal Notification Template Variables documentation (#23031)
DataDog/documentation: content/en/security/notifications/variables.md

To see a signal's list of event attributes, click **JSON** at the bottom of the **Overview** tab in the signal's side panel. Use the following syntax to add these event attributes in your rule notifications: `{{@attribute}}`. To access inner keys of the event attributes, use JSON dot notation, for example, `{{@attribute.inner_key}})`.

> If the signal JSON does not contain an attribute which is present in the related log JSON, use the previously outlined syntax with the attribute name from the log JSON. This attribute will then be included in both the signal JSON and the signal notifications.

Reviewer: ```suggestion
If the signal's JSON does not contain an attribute that is present in the related log's JSON, use the previously outlined syntax with the attribute name from the log's JSON. This attribute is then included in both the signal's JSON and the signal notifications.
```

Author: Thank, I did the update in this commit: https://github.com/DataDog/documentation/pull/23031/commits/ce3b83b723997b3d7332167376d8a83820479a0b
